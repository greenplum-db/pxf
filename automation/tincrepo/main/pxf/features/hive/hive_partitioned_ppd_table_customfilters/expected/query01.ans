-- start_ignore
-- end_ignore
-- @description query01 for PXF Hive partitioned table with custom filters
SELECT * FROM pxf_hive_partitioned_ppd_table_customfilter_1 ORDER BY n1, s2;
  s1  | d1 |  s2  | n1
------+----+------+----
 row9 | 14 | s_14 |  9
(1 row)

SELECT * FROM pxf_hive_partitioned_ppd_table_customfilter_2 ORDER BY n1, s2;
  s1  | d1 | s2  | n1
------+----+-----+----
 row2 |  7 | s_7 |  2
(1 row)

SELECT * FROM pxf_hive_partitioned_ppd_table_customfilter_3 ORDER BY n1, s2;
  s1   | d1 |  s2  | n1
-------+----+------+----
 row1  |  6 | s_6  |  1
 row2  |  7 | s_7  |  2
 row3  |  8 | s_8  |  3
 row4  |  9 | s_9  |  4
 row5  | 10 | s_10 |  5
 row6  | 11 | s_11 |  6
 row7  | 12 | s_12 |  7
 row8  | 13 | s_13 |  8
 row9  | 14 | s_14 |  9
 row10 | 15 | s_15 | 10
(10 rows)

