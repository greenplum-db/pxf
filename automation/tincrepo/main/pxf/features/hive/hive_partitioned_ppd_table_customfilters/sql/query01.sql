-- @description query01 for PXF Hive partitioned table with custom filters

SELECT * FROM pxf_hive_partitioned_ppd_table_customfilters ORDER BY n1, s2;