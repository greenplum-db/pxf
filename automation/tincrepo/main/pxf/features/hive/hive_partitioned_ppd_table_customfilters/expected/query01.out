-- @description query01 for PXF Hive partitioned table with custom filters
SELECT * FROM pxf_hive_partitioned_ppd_table_customfilter_1 ORDER BY num1, t2;
  t1  | dub1 |  t2  | num1 
------+------+------+------
 row9 |   14 | s_14 |    9
(1 row)

SELECT * FROM pxf_hive_partitioned_ppd_table_customfilter_2 ORDER BY num1, t2;
  t1  | dub1 | t2  | num1 
------+------+-----+------
 row2 |    7 | s_7 |    2
(1 row)

SELECT * FROM pxf_hive_partitioned_ppd_table_customfilter_3 ORDER BY num1, t2;
  t1   | dub1 |  t2  | num1 
-------+------+------+------
 row1  |    6 | s_6  |    1
 row2  |    7 | s_7  |    2
 row3  |    8 | s_8  |    3
 row4  |    9 | s_9  |    4
 row5  |   10 | s_10 |    5
 row6  |   11 | s_11 |    6
 row7  |   12 | s_12 |    7
 row8  |   13 | s_13 |    8
 row9  |   14 | s_14 |    9
 row10 |   15 | s_15 |   10
(10 rows)

SELECT * FROM pxf_hive_partitioned_ppd_table_customfilter_4 ORDER BY num1, t2;
  t1   | dub1 |  t2  | num1 
-------+------+------+------
 row1  |    6 | s_6  |    1
 row2  |    7 | s_7  |    2
 row3  |    8 | s_8  |    3
 row4  |    9 | s_9  |    4
 row5  |   10 | s_10 |    5
 row6  |   11 | s_11 |    6
 row7  |   12 | s_12 |    7
 row8  |   13 | s_13 |    8
 row10 |   15 | s_15 |   10
(9 rows)

